### PR TITLE
{tools}[foss/2022b] LASSO-Python v2.0.0 w/ Python 3.10.8

### PR DESCRIPTION
**Pull request by VRehnberg**
(created using `eb --new-pr`)
requires:
 - https://github.com/easybuilders/easybuild-easyconfigs/pull/17453

### CODE REVIEW

File: easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('mdurl', '0.1.2', {
+        'checksum': 'bb413d29f5eea38f31dd4754dd7377d4465116fb207585f97bf925588687c1ba',
+    }),
+    ('markdown-it-py', '2.2.0', {
+        'modulename': 'markdown_it',
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb:31]: remove trailing whitespace
```suggestion
        'modulename': 'markdown_it',
```

File: easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        'checksum': '7c9a5e412688bc771c67432cbfebcdd686c93ce6484913dccf06cb5a0bea35a1',
+    }),
+    ('rich', '12.6.0', {
+        'checksum': '125d96d20c92b946b983d0d392b84ff945461e5a06d3867e9f9e575f8697b67f',
+    }),
+    (name, version, {
+        'sources': {
+            'filename': 'lasso_python-%(version)s.tar.gz',
+            'download_filename': 'lasso_python-%(version)s.tar.gz',
+        },
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb:41]:
```suggestion
        'source_tmpl': 'lasso_python-%(version)s.tar.gz',
```

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
node078.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/08f2b16bfbd1fe55b53de16d8445a1dc for a full test report.

File: easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    ('matplotlib', '3.7.0'),
+    ('scikit-learn', '1.2.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('mdurl', '0.1.2', {
+        'checksum': 'bb413d29f5eea38f31dd4754dd7377d4465116fb207585f97bf925588687c1ba',
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb:28]:
```suggestion
        'checksum': ['bb413d29f5eea38f31dd4754dd7377d4465116fb207585f97bf925588687c1ba'],
```

File: easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    }),
+    ('markdown-it-py', '2.2.0', {
+        'modulename': 'markdown_it',
+        'checksum': '7c9a5e412688bc771c67432cbfebcdd686c93ce6484913dccf06cb5a0bea35a1',
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb:32]:
```suggestion
        'checksum': ['7c9a5e412688bc771c67432cbfebcdd686c93ce6484913dccf06cb5a0bea35a1'],
```

File: easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    }),
+    ('rich', '12.6.0', {
+        'checksum': '125d96d20c92b946b983d0d392b84ff945461e5a06d3867e9f9e575f8697b67f',
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb:35]:
```suggestion
        'checksum': ['125d96d20c92b946b983d0d392b84ff945461e5a06d3867e9f9e575f8697b67f'],
```

File: easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    }),
+    (name, version, {
+        'sources': {
+            'filename': 'lasso_python-%(version)s.tar.gz',
+            'download_filename': 'lasso_python-%(version)s.tar.gz',
+        },
+        'modulename': 'lasso',
+        'checksum': '67776e1876764680ac179fb750695aac63ecdbdf4f1ac8439f7e76fa3956727c',
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb:43]:
```suggestion
        'checksum': ['67776e1876764680ac179fb750695aac63ecdbdf4f1ac8439f7e76fa3956727c'],
```

**Review by jfgrimm — DISMISSED** [review]
make CI happy by having checksums in lists

File: easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        'modulename': 'markdown_it',
+        'checksum': ['7c9a5e412688bc771c67432cbfebcdd686c93ce6484913dccf06cb5a0bea35a1'],
+    }),
+    ('rich', '12.6.0', {
+        'checksum': ['125d96d20c92b946b983d0d392b84ff945461e5a06d3867e9f9e575f8697b67f'],
+    }),
+    (name, version, {
+        'source_tmpl': 'lasso_python-%(version)s.tar.gz',
+        'modulename': 'lasso',
+        'checksum': ['67776e1876764680ac179fb750695aac63ecdbdf4f1ac8439f7e76fa3956727c'],
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/LASSO-Python/LASSO-Python-2.0.0-foss-2022b.eb:40]: missed the typo
```suggestion
exts_list = [
    ('mdurl', '0.1.2', {
        'checksums': ['bb413d29f5eea38f31dd4754dd7377d4465116fb207585f97bf925588687c1ba'],
    }),
    ('markdown-it-py', '2.2.0', {
        'modulename': 'markdown_it',
        'checksums': ['7c9a5e412688bc771c67432cbfebcdd686c93ce6484913dccf06cb5a0bea35a1'],
    }),
    ('rich', '12.6.0', {
        'checksums': ['125d96d20c92b946b983d0d392b84ff945461e5a06d3867e9f9e575f8697b67f'],
    }),
    (name, version, {
        'source_tmpl': 'lasso_python-%(version)s.tar.gz',
        'modulename': 'lasso',
        'checksums': ['67776e1876764680ac179fb750695aac63ecdbdf4f1ac8439f7e76fa3956727c'],
```

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node078.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/787982fc1ba59498a90f6e45a88d22b9 for a full test report.

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @VRehnberg!

**Pull request merged**